packages/web/src/components/molecules/Console/ConsoleLines.tsx

```diff
@@ -9,11 +9,23 @@ export const ConsoleLines: FC<{
   start: number;
   maxDigits: number;
   LineComponent: ConsoleProps['LineComponent'];
-}> = memo(({lines, start, maxDigits, LineComponent = ConsoleLine}) => (
+  onSelectLine?: (line: number) => void;
```

Comment:
It's `onClick` actually, the line shouldn't necessary now that it selecting, but knows that it's clicked